testcontainers/src/images/local_stack.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const NAME: &str = "localstack/localstack";
+const TAG: &str = "latest";
```

Comment:
```suggestion
const TAG: &str = "1.4";
```

Pin the version, maybe even to `1.4.0`.

Reply:
agree